docs/logs/customizing-the-sdk/README.md

```diff
@@ -14,16 +14,25 @@ TODO
 
 ### IncludeScopes
 
-A "[scope](https://docs.microsoft.com/dotnet/core/extensions/logging#log-scopes)"
-is an `ILogger` concept that can group a set of logical operations and attach
-data to each log created as part of a set.
+Log
+[scope](https://docs.microsoft.com/dotnet/core/extensions/logging#log-scopes) is
+an `ILogger` concept that can group a set of logical operations and attach data
+to each log created as part of a set.
 
 `IncludeScopes` is off by default. Setting this to `true` will include all
```

Comment:
leaving a comment for a potential follow up PR: Warn that `IncludeScopes` will enable scopes from all "depth/level", and there is no fine grained control over it. A lot of times, framework itself adds information to Scopes, that are duplicate of information already available in OTel.